src/main/java/org/sigmah/server/service/UserPermissionPolicy.java

```diff
@@ -228,4 +238,24 @@ public boolean isGranted(final OrgUnitProfile userOrgUnit, final GlobalPermissio
 		return false;
 	}
 
+	public boolean isGranted(List<OrgUnitProfile> userUnits, GlobalPermissionEnum permission) {
```

Comment:
@numero-six It is usually the latter. I love `final` but we didn't really enforce its usage so a lot of methods are missing it.